pagerduty/data_source_pagerduty_extension_schema.go

```diff
@@ -0,0 +1,75 @@
+package pagerduty
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/heimweh/go-pagerduty/pagerduty"
+)
+
+func dataSourcePagerDutyExtensionSchema() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourcePagerDutyExtensionSchemaRead,
+
+		Schema: map[string]*schema.Schema{
+			"name_regex": {
```

Comment:
If I had to guess, I would think`name_regex` here was copied from somewhere? Perhaps PagerDutyVendor data source, where I also see it. It should be removed here as it's not used.